checks/0B2F82.yaml

```diff
@@ -0,0 +1,43 @@
+id: "0B2F82"
+name: SAPHanaSR-angi version identical on all nodes
+group: OS and package versions
+description: |
+  Installed SAPHanaSR-angi version is identical on all nodes
+remediation: |
+  ## Abstract
+  Installed SAPHanaSR-angi version must be identical on all nodes of the cluster
+
+  ## Remediation
+  Install identical SAPHanaSR-angi version on all nodes of the cluster
+
+  ## Reference
```

Comment:
Do we need this empty `reference`?
I would remove it if we don't have any..
@abravosuse what do you think?